subprojects/tooling-api/src/main/java/org/gradle/tooling/LongRunningOperation.java

```diff
@@ -130,7 +130,7 @@ public interface LongRunningOperation {
      * System properties can be also defined in the build scripts (and in the gradle.properties file), or with a JVM argument. In case of an overlapping system property definition the precedence is as follows:
      * <ul>
      *     <li>{@code withSystemProperties()} (highest)</li>
-     *     <li>{@code withJvmArguments()}</li>
+     *     <li>{@code addJvmArguments()} and {@code setJvmArguments()}</li>
```

Comment:
```suggestion
     *     <li>{@code addJvmArguments(...)} and {@code setJvmArguments(...)}</li>
```

Reply:
There are two methods with `String,..` and `Iterable<String>` parameter types, I would use a `...` notation.

Reply:
You are right; I've adjusted the comment.